File: content/fixtures/code.java
import java.util.List;

class Code {
    record Task(String area, String owner, String detail) {}

    private static List<String> describe(List<Task> tasks) {
        return tasks.stream()
            .map(task -> task.area() + " assigned to " + task.owner() + ": " + task.detail())
            .toList();
    }

    public static void main(String[] args) {
        var tasks = List.of(
            new Task("catalog", "Iris", "verify product groups"),
            new Task("pricing", "Noah", "extraction fixture marker: shared code sample"),
            new Task("publishing", "Maya", "prepare release notes")
        );

        describe(tasks).forEach(System.out::println);
    }
}

File: content/fixtures/code.rs
struct Finding {
    area: &'static str,
    severity: &'static str,
    detail: &'static str,
}

fn describe(finding: &Finding) -> String {
    format!("{} [{}]: {}", finding.area, finding.severity, finding.detail)
}

fn main() {
    let findings = [
        Finding { area: "layout", severity: "low", detail: "spacing checked" },
        Finding { area: "copy", severity: "medium", detail: "extraction fixture marker: shared code sample" },
        Finding { area: "links", severity: "low", detail: "anchors valid" },
    ];

    for finding in findings.iter() {
        println!("{}", describe(finding));
    }
}
